opengrok-indexer/src/test/java/org/opengrok/indexer/search/SearchEngineTest.java

```diff
@@ -148,6 +156,69 @@ void testGetQuery() throws Exception {
                 instance.getQuery());
     }
 
+    @Test
+    void testSortOrderLastModified() {
+        SearchEngine instance = new SearchEngine();
+        instance.setFile("main.c");
+        instance.setFreetext("arguments");
+        instance.setSortOrder(SortOrder.LASTMODIFIED);
+        int hitsCount = instance.search();
+        List<Hit> hits = new ArrayList<>();
+        instance.results(0, hitsCount, hits);
+        assertTrue(hits.size() > 1, "Should return at least 2 hits for RELEVANCY sort to check order");
```

Comment:
Why not check the exact count ? If count is different than the array length, it does not make sense to proceed with comparison.